README.md

```diff
@@ -0,0 +1,220 @@
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [ClinixGo] <a name="about-project"></a>
+
+> The project is based on an app to book an appointment with a health specialist(Doctor). The doctor appointment booking app permits healthcare providers to manage their appointments with increased efficiency while giving reliable management. However, it displays all details of the patients, complete history of appointment, analyses the doctors, saving time, etc.
+
+**[ClinixGo]** is an appointment booking app
+
+### ERD Diagram for Project
+
+<img src="https://res.cloudinary.com/dxsom7jmx/image/upload/v1678138506/drawSQL-final-capstoneclinixgo-export-2023-03-06_1_wxo60f.png" width="1000px" alt="ERD Diagram">
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Blog App is backend project
+
+<details>
+<summary>Front-End</summary>
+  <ul>
+    <li><a href="https://redux.js.org/">Redux</a></li>
+    <li><a href="https://reactjs.org/">ReactJs</a></li>
+    <li><a href="https://webpack.js.org/">Webpack</a></li>
+    <li><a href="https://getbootstrap.com/">Bootstrap</a></li>
+    <li><a href="https://react-icons.github.io/react-icons/">ReactIcon</a></li>
+    <li><a href="https://reactjs.org/">PrimeReact</a></li>
+    <li><a href="https://reactjs.org/">ReactRouter</a></li>
+  </ul>
+</details>
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://jwt.io/">Jwt</a></li>
+    <li><a href="https://github.com/rswag/rswag">rswag</a></li>
+    <li><a href="https://github.com/CanCanCommunity/cancancan">Cancancan</a></li>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+    <li><a href="https://swagger.io/tools/swagger-ui/">Swagger-Ui</a></li>
+    <li><a href="https://rubyonrails.org/">Ruby On Rails</a></li>
+
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> We can do CRUD operations on following things:
+
+- **Doctors**
+- **Appoitnments**
+- **Users**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Coming Soon 🎈]()
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> In order to start with this project do as follows :
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+`gem install rails`
+
+### Setup
+
+Clone this repository to your desired folder:
+
+`git clone git@github.com:Strangeal/ClinixGo.git`
+
+then
+
+`cd ClinixGo`
+
+### Install
+
+Install this project dependencies with:
+
+`bundle install` <br />
+`npm install`
+
+### Usage
+
+`./bin/dev`
+
+### Run Test
+
+To run app test. run the following commad:
+
+`rspec`
```

Comment:
- [ ] If I run your tests according to your instructions, I get this:

![image](https://user-images.githubusercontent.com/57421823/223276023-16e7440e-ef07-476f-bd1a-f77da90bd64a.png)

If these tests are passing for you, kindly add instructions on your readme about how to properly run them and make them pass.  If this is a bug in the tests, kindly fix it to ensure proper functionality. 👍 

Also, there are 2 tests that are "pending", kindly either remove it or implement it.+

That way, your app's integrity will be secured.